doc/posts/2023/2023_05_19_vara_week0.rst

```diff
@@ -0,0 +1,34 @@
+GSOC 2023 week 0 blog
```

Comment:
please, improve title.  Not enough explicit, too general

Also, add title marker below `====`